lib/assets/map_cell/main.js

```diff
@@ -546,4 +550,19 @@ export function init(ctx, payload) {
       idx: idx && parseInt(idx),
     });
   }
+
+  function updateCenter(value) {
+    const coordinates = value.match(/(-?\d+\.?\d*),\s*(-?\d+\.?\d*)/);
```

Comment:
If there are no characters it's likely not a search, so we can use a less strict check

```suggestion
    const coordinates = value.match(/[\d\.\s]+/);
```

Reply:
It can be a postal code. They’re also supported as queries, with or without separators.